src/components/loader/page.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Button from '../button';
+
+type PropTypes = {
+  backgroundColor?: string;
+  height?: number;
+  reset?: any;
+};
+
+export const Loading: React.FC<PropTypes> = () => {
+  return (
+    <div className="absolute w-full h-full flex justify-center">
+      <div className=" flex justify-center flex-col items-center">
+      <p>Chargement ...</p>
```

Comment:
use "Loading" as an English word instead of "chargement ". For good practice, I recommend using only one language in the project unless the team has agreed to use multiple languages.